### PR TITLE
Converse

### LINKED ISSUE #16: Re-implement Converse

### PR DESCRIPTION
**Pull request by revangel**
# Overview

This MR implements the **converse** action

### CODE REVIEW

File: nb2/bot/slack_bot.py [hunk 1]
```diff
@@ -246,6 +249,53 @@ def random(self):
...
+        ]
+
+        if any(people_with_no_quotes):
+            missing_people = ", ".join(people_with_no_quotes[:-1])
+            message = f"I don't recognize {missing_people}"
+            return self.Result(ok=False, message=message)
+
+        message = ""
+
+        for i in range(0, QUOTES_PER_PERSON):
```
Comment (harry-kim) [inline nb2/bot/slack_bot.py]: range starts at 0 by default

File: tests/bot/test_slackbot.py [hunk 2]
```diff
@@ -277,6 +277,38 @@ def test_random_responds_with_random_quote(client, session, mock_bot):
...
+    session.bulk_save_objects([mock_person1, mock_person2, *mock_quotes1, *mock_quotes2])
+    session.commit()
+
+    result = mock_bot.converse([mock_person1.slack_user_id, mock_person2.slack_user_id])
+    message = result.message
+
+    assert message.count("Beth:") == 2
+    assert message.count("Vasily:") == 2
+
+
```
Comment (harry-kim) [inline tests/bot/test_slackbot.py:348]: can we test when a user has no quotes?

File: nb2/bot/slack_bot.py (hunk 1 above)
Comment (revangel) [reply nb2/bot/slack_bot.py]: Good catch - removed

File: tests/bot/test_slackbot.py (hunk 2 above)
Comment (revangel) [reply tests/bot/test_slackbot.py:348]: Added!

**Comment by harry-kim** [comment]
+1 !

**Pull request merged**